cts/cli/regression.tools.exp

```diff
@@ -30,10 +30,10 @@ Also known as properties, these are options that affect behavior across the enti
 
   * dc-version: Pacemaker version on cluster node elected Designated Controller (DC)
     * Includes a hash which identifies the exact revision the code was built from. Used for diagnostic purposes.
-    * Allowed values: string (default: )
+    * Allowed values: (generated by Pacemaker) string (default: )
```

Comment:
Maybe "Possible values (generated by Pacemaker): string (default: none)"?

Reply:
That's probably better. I don't like having `(generated by Pacemaker)` right in front of the option list, because it makes it look like it's associated with the first option. Likewise with putting it after the option list.